lib/fathead/android_api_docs/parse.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin env python
+# -*- coding: utf-8 -*-
+from bs4 import BeautifulSoup
+from glob import glob
+import re
+
+def build_article(tr):
+    '''Builds fathead article entry.
+
+    '''
+
+    title = tr.td.a.contents[0]
+    url = (tr.find('a')['href'])
+    #Some packages don't have a description, but simply a page listing classes and interfaces (possibly enums)
+    try:
+        abstract = tr.find('p').contents[0]
+    except:
+        abstract = "More information about interfaces and classes at " + url
+    title = '<span class="prog__sub">%s</span>' % title
+    #Replace newline characters with a space
+    abstract = re.sub('\n', ' ', abstract)
```

Comment:
@TM37 as long as the text isn't code (meaning inside a `<pre><code>` then yes we'll wrap the text.

We also force the height of the initial display to a fixed height so that the IA can't fill the page with its content until the user clicks to expand. If the original content has newlines I think we should keep them. If there are extra newlines, we can also reduce them to a single newline: `re.sub('\n+', '\\n', abstract)`